ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Mock/Block/PraosRule.hs

```diff
@@ -83,8 +83,8 @@ instance SimpleCrypto c
 
 instance SimpleCrypto c
       => ProtocolLedgerView (SimplePraosRuleBlock c) where
-  protocolLedgerView _ _ = ()
-  anachronisticProtocolLedgerView _ _ _ = Just $ SB.unbounded ()
+  protocolLedgerView _ _ = error "TODO"
+  anachronisticProtocolLedgerView _ _ _ = Just $ SB.unbounded $ error "TODO"
```

Comment:
This still needs to be fixed.

Reply:
In what sense? If the ledger view is simply `()`, then this seems entirely correct.

Reply:
Unfortunately, the `LedgerView` is no longer `()`, because to implement `protocolSlotLength :: NodeConfig p -> LedgerView p -> SlotLength` in `WithLeaderSchedule`, we need the underlying `LedgerView p` when forwarding `protocolSlotLength` to the underlying protocol (so we can't set it to `()`).